src/main/java/com/epam/training/auction_backend/entity/Auction.java

```diff
@@ -0,0 +1,132 @@
+package com.epam.training.auction_backend.entity;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "AUCTIONS")
+public class Auction {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "ID", unique = true, nullable = false)
+	private long id;
+	
+	@Column(name = "TITLE")
+	private String title;
+	
+	@Column(name = "DESCRIPTION")
+	private String description;
+	
+	@Column(name = "SELLER_USER_ID")
+	private long sellerUserId;
+	
+	@Column(name = "WINNER_USER_ID")
+	private long winnerUserId;
+	
+	@Column(name = "FINAL_PRICE")
+	private double finalPrice;
+	
+	@Column(name = "IS_ACTIVE")
+	private boolean isActive;
+	
+	@Column(name = "CREATED_AT")
+	private LocalDateTime createdAt;
+	
+	@Column(name = "UPDATED_AT")
+	private LocalDateTime updatedAt;
+
+	public Auction() {
+	}
+	
+	public Auction(String title, String description, long sellerUserId) {
```

Comment:
I think you should be passing a who User object for the seller instead of the id only.